### DISCUSSION TITLE
curl configure cannot find openssl

### DISCUSSION THREAD

**Discussion by neolib** [msys2/MINGW-packages#8788] [General]
I am having a weird problem with curl configure.

I ran the following command in mingw32:
`configure --with-openssl`
then `make` but always get:
```
configure: OPT_OPENSSL: yes
configure: OPENSSL_ENABLED:
configure: error: --with-openssl was given but OpenSSL could not be detected
```

I already have openssl-devel installed:
```
$ pacman -Q openssl-devel
openssl-devel 1.1.1.k-1
```
Did anyone have a similar problem?

**Comment by Biswa96** [comment]
As you have opened this discussion in MINGW-package repository, I assume you are using mingw program. If yes you need to install package with `$MINGW_PACKAGE_PREFIX`. For example, `pacman -S --needed mingw-w64-x86_64-openssl`.

**Reply by neolib** [reply]
Okay... That's it! This is not the first time I messed up packages in MSYS2 and MINGW...

**Reply by Biswa96** [reply]
If you have more questions feel free to ask it any time or join msys2 channel in Gitter or Discord for further discussion.
